src/sentry/shared_integrations/client/base.py

```diff
@@ -376,6 +376,31 @@ def record_response(self, response: Response):
                     return
                 if is_response_error(response):
                     buffer.record_error()
+                if randint(0, 99) == 0:
+                    (
+                        rpc_integration,
+                        rpc_org_integration,
+                    ) = integration_service.get_organization_contexts(
+                        integration_id=self.integration_id
+                    )
+                    if rpc_integration.provider == "github" or "gitlab":
```

Comment:
@chloeho7 this doesn't do what you think it does lol

Reply:
This can be 

`if rpc_integration.provider in ("github", "gitlab")`